packages/admin/cms-admin/src/common/header/Header.sc.ts

```diff
@@ -0,0 +1,18 @@
+import { Box, Button as MUIButton } from "@mui/material";
```

Comment:
*.sc.ts files are considered to be "private" files of the respective components and should not be used across multiple components.